src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -159,7 +164,46 @@ public static void init() {
         }
 
         for (ItemStack ore : OreDictionary.getOres("oreQuartz")) {
-            RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), new ItemStack(Items.QUARTZ, 6));
+            RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1),
+                  StackUtils.size(OreDictionary.getOres("gemQuartz").get(0), 6));
```

Comment:
please pull all `OreDictionary.getOres` calls out of the loops in this block

Reply:
also please remove all the try/catch blocks where the catch is empty, they're bad legacy code

Reply:
to your first comment: yes. i just did that bad.
to your second one: ok i will remove them, but i didn't made them all.